rootfs/api/models.py

```diff
@@ -881,7 +891,8 @@ def publish(self, source_version='latest'):
             source_image = "{}:{}".format(source_image, source_tag)
         # If the build has a SHA, assume it's from deis-builder and in the deis-registry already
         deis_registry = bool(self.build.sha)
-        publish_release(source_image, self.config.values, self.image, deis_registry)
+        if self.build.dockerfile:
+            publish_release(source_image, self.config.values, self.image, deis_registry)
```

Comment:
why wouldn't we want to publish a release if no dockerfile is present? Logically speaking, any build we create should be published.